.github/workflows/bump-cli.yml

```diff
@@ -1,35 +1,32 @@
 name: Bump CLI version
 on:
   workflow_dispatch:
-  pull_request:
-    branches: [main]
-    paths:
-      - .github/actions/create-pr/action.yml
-  schedule:
-  - cron: 0 0 */14 * * # run every 14 days
 
 jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    environment: cli-updates
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 1
-          token: ${{ secrets.WORKFLOW_TOKEN }}
       - name: Bump CLI
         env:
           GH_TOKEN: ${{ github.token }}
         run: |
           scripts/replace-cli-version.sh
-      - name: Commit, Push and Open a PR
-        uses: ./.github/actions/create-pr
+      - name: Create the PR
+        uses: koesie10/create-app-pr-action@v1
```

Comment:
I didn't go this route because of this dependency on external accounts. I don't mind it if we are ok with this 😄 but I think usually we vendor all our stuff or at the very least have them under the `github` org

Reply:
It might be possible to vendor this action since it's just a `Dockerfile` and some Go files. That might be the best way to go forward with this if we do decide to go with this approach.